vcpkg-artifacts/vcpkg.ts

```diff
@@ -44,6 +44,16 @@ export function vcpkgFetch(session: Session, fetchKey: string): Promise<string>
   });
 }
 
+export function vcpkgExtract(session: Session, archive: string, target:string, strip:string): Promise<string> {
+  return runVcpkg(session.vcpkgCommand, ['z-extract', archive, target, strip]).then((output) => {
+    return output;
+  }, (error) => 
+ {
+    session.channels.warning('Failed to extract: ' + archive + '\n');
```

Comment:
This needs localization. But I think just emitting what vcpkg.exe said is fine. Note that I couldn't do that for the download one because `--machine-readable-progress`.